tests/helpers/browser-detect.js

```diff
@@ -1,4 +1,7 @@
 // `window.ActiveXObject` returns undefined in IE11 (as well as non-IE browsers)
 // `"ActiveXObject" in window` returns `true` in all IE versions
 // only IE11 will pass _both_ of these conditions
-export default !window.ActiveXObject && 'ActiveXObject' in window;
+export const isIE11 = !window.ActiveXObject && 'ActiveXObject' in window;
```

Comment:
should we explicitly check for `undefined` instead of falsyness?

Reply:
I can try that, will confirm in a local VM...

Reply:
LOLOLOL, I tested changing this.  tldr IE11 makes no sense 🙃 

`window.ActiveXObject` is actually a function (not `undefined` or `false`), but `!!window.ActiveXObject` is false 😝 ...